docs/resources/private_endpoint_connection.md

```diff
@@ -16,14 +16,14 @@ Private endpoint connections allow customer applications to connect to a Cockroa
 ## Example with AWS PrivateLink
 
 # Enable the PrivateLink service on the CockroachDB Cloud cluster.
-resource "cockroach_private_endpoint_services" "services" {
+resource "cockroach_private_endpoint_services" "aws_cluster" {
```

Comment:
IMO, I find `aws_cluster` name for the services more confusing that `services`

Reply:
I renamed here because I felt that the repetition of the resource name and the name of the attribute led to the original error.   Maybe @marksoper has a thought here?

Reply:
I talked to mark about this.  We were thinking a happy middleground is to name these like `aws_cluster_services`